From 6e783b40766fb6664d3c49332da7a1cadfebfbc5 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 9 Dec 2013 13:57:35 +0100
Subject: [PATCH 142/212] fix intel-hda live migration

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1386597455-25593-2-git-send-email-kraxel@redhat.com>
Patchwork-id: 56035
O-Subject: [RHEL-7 qemu-kvm PATCH 1/1] fix intel-hda live migration
Bugzilla: 1036537
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>

RHEL-6 intel-hda emulation doesn't support MSI, so we have
to turn it off for thel RHEL-6 machine types in RHEL-7.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 hw/i386/pc_piix.c | 4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/i386/pc_piix.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 3704933..189d2fa 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -1012,6 +1012,10 @@ static QEMUMachine pc_machine_rhel700 = {
         .property = "use_broken_id",\
         .value    = stringify(1),\
     },{\
+        .driver   = "intel-hda",\
+        .property = "msi",\
+        .value    = stringify(0),\
+    },{\
         .driver = "qemu32-" TYPE_X86_CPU,\
         .property = "xlevel",\
         .value = stringify(0),\
-- 
1.7.1

